services/frontend-service/src/ui/components/SideBar/SideBar.tsx

```diff
@@ -136,6 +140,25 @@ export const getActionDetails = (
                 lockMessage: appLocks.find((lock) => lock.lockId === action.deleteEnvironmentApplicationLock.lockId)
                     ?.message,
             };
+        case 'deleteEnvironmentTeamLock':
+            return {
+                type: ActionTypes.DeleteEnvironmentTeamLock,
+                name: 'Delete Team Lock',
+                dialogTitle: 'Are you sure you want to delete this team lock?',
+                summary:
+                    'Delete team lock for "' +
+                    action.deleteEnvironmentTeamLock.team +
+                    '" on ' +
+                    action.deleteEnvironmentTeamLock.environment +
+                    ' with the message: "' +
+                    teamLocks.find((lock) => lock.lockId === action.deleteEnvironmentTeamLock.lockId)?.message +
+                    '"',
+                tooltip: 'This will only remove the lock, it will not automatically deploy anything.',
+                environment: action.deleteEnvironmentTeamLock.environment,
+                team: action.deleteEnvironmentTeamLock.team,
+                lockId: action.deleteEnvironmentTeamLock.lockId,
+                lockMessage: teamLocks.find((lock) => lock.lockId === action.deleteEnvironmentTeamLock.lockId)?.message,
```

Comment:
the lock Id may be the same for 2 different team locks, if they have different environments.
So I think this comparison here may not be correct.

Reply:
Isn't the lock id supposed to be unique?

Reply:
2 Team locks for the same team but different environments are different locks alltogether

Reply:
afair, we don't make any effort to make them truly unique (this would have to happen when creating any lock id). And since they are all stored in different directories, there's no inherent requirement for them to be globally unique.